gnovm/pkg/repl/repl.go

```diff
@@ -0,0 +1,270 @@
+package repl
+
+import (
+	"bytes"
+	"fmt"
+	"go/ast"
+	"go/format"
+	"go/parser"
+	"go/printer"
+	"go/token"
+	"text/template"
+
+	gno "github.com/gnolang/gno/gnovm/pkg/gnolang"
+	"github.com/gnolang/gno/gnovm/tests"
```

Comment:
```suggestion
```

Reply:
this is needed to use the previously used test storage.